frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tests.tsx

```diff
@@ -1400,40 +1396,4 @@ describe("Activity Feed", () => {
       )
     ).toBeInTheDocument();
   });
-
-  it("renders setup experience installed software correctly", () => {
```

Comment:
wait, why are these removed?

Reply:
These were testing that if the author data was not included for these activities then `Fleet` would be displayed as the actor name, but this is not the case anymore. The activities API will send back the name `Fleet` in the author_name so these test would be both be incorrect and not needed.